### PR TITLE
Update Ohai Custom Plugins

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kgarmoe@chef.io>

## Description

Clarifies where to store custom Ohai plugin code. Adds shortcodes, edits related content, and fixes markdown, too.

## Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

**Review by teknofire — APPROVED** [review]
Looks great to me!

File: content/ohai.md [hunk 1]
```diff
@@ -16,12 +16,9 @@ aliases = ["/ohai.html"]
 
 {{% ohai_summary %}}
 
-Ohai collects data for many platforms, including AIX, Darwin, Linux,
-FreeBSD, OpenBSD, NetBSD, Solaris, and any Microsoft Windows operating
-systems.
+Ohai collects data for many platforms, including AIX, Darwin, Linux, FreeBSD, OpenBSD, NetBSD, Solaris, and any Microsoft Windows operating systems.
```
Comment (tas50) [inline content/ohai.md]:
```suggestion
Ohai collects data for many platforms, including AIX, macOS, Linux, FreeBSD, Solaris, and any Microsoft Windows operating systems.
```

File: content/ohai.md [hunk 2]
```diff
@@ -196,21 +191,19 @@ windows
...
+Ohai ships several optional plugins that you can enable in the [client.rb configuration file](/config_rb_client/).
 
 - `:IPC` - SysV IPC shmem information (New in Chef Infra Client 16)
 - `:Interupts` - Data from /proc/interrupts and /proc/irq (New in Chef Infra Client 16)
 - `:Lspci` - PCI device information on Linux hosts.
 - `:Lsscsi` - SCSI device information on Linux hosts.
-- `:Passwd` - User and Group information on non-Windows hosts. This plugin can result in very large node sizes if a system connects to Active Directory or LDAP.
+- `:Passwd` - User and Group information on non-Windows hosts. This plugin can result in large node sizes if a system connects to Active Directory or LDAP.
 - `:Sessions` - Sessions data from loginctl on Linux hosts.
 - `:Sysctl` - All sysctl values on Linux hosts.
```
Comment (tas50) [inline content/ohai.md:203]: We're missing a one more here:

Grub2: Information from the Grub2 bootloader on Linux

File: content/ohai.md [hunk 3]
```diff
@@ -266,14 +250,11 @@ else
 end
 ```
 
-Hint files are located in the `/etc/chef/ohai/hints/` directory by
-default. Use the `Ohai.config[:hints_path]` setting in the [client.rb
-configuration file](/config_rb_client/) to customize this location.
+Hint files are located in the `/etc/chef/ohai/hints/` directory by default. Use the `Ohai.config[:hints_path]` setting in the [client.rb configuration file](/config_rb_client/) to customize this location.
 
 ## ohai Cookbook Resource
```
Comment (tas50) [inline content/ohai.md]:
```suggestion
## ohai Resource
```

**Review by tas50 — CHANGES_REQUESTED** [review]
I left a few suggestions

**Review by tas50 — APPROVED** [review]
ship it

**Review by jonlives — APPROVED** [review]

**Comment by IanMadd** [comment]
I'm surprised the linter isn't freaking out about `themes/docs-new/layouts/shortcodes/cookbooks_content.md`, but I guess it doesn't check shortcodes.

**Review by IanMadd — APPROVED** [review]

**Comment by kagarmoe** [comment]
@IanMadd I expected it to blow up, too.

**Pull request merged**